apps/ml/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+services:
+  core_api:
+    build: .
+    container_name: "ml-api-container"
+    ports:
+      - "8000:80"
+    volumes:
+      - ./app/:/app
+    command: uvicorn app.main:app --host 0.0.0.0 --port 15400 --reload
```

Comment:
This won't work, we changed the Dockerfile to port 80 and here you bind 8000:80 so the uvicorn must be running on port 80 😄 . So we need to change `15400` to `80`